### ISSUE TITLE
Warning when including BiocManager code in vignette

### ISSUE THREAD

**Issue by lmweber** [Bioconductor/BiocCheck#152]
Currently if any `BiocManager` code is included in a package vignette, `BiocCheck` returns a WARNING, e.g. as follows:

```
* Checking vignette directory...
    * WARNING: BiocInstaller code found in vignette(s)
      Found in file(s):
        nnSVG.Rmd (line 39)
```

I have tried this both with `eval=FALSE` in the RMarkdown code block, and with commenting out the `BiocManager` line inside the code block.

Possibly `BiocCheck` is noticing any string with `BiocManager` included in the vignette, although I'm not completely sure.

We noticed this during my recent package submission at https://github.com/Bioconductor/Contributions/issues/2543. (There are some links to build reports with additional details in this issue.)

**Comment by LiNk-NY** [comment]
Hi Lukas, @lmweber 

Thank you for reporting.
This has been fixed in the branch of `BiocCheck` that will soon be merged to the default branch.
If you'd like to test, see the `output` branch. 

Best,
Marcel

**Comment by lmweber** [comment]
Thank you!

**Comment by LiNk-NY** [comment]
Closed in 16cf48c667858f9ce7dadefe8f69140c9efcfd91 1.31.37
